stable/prometheus-operator/templates/exporters/kube-api-server/servicemonitor.yaml

```diff
@@ -14,6 +14,10 @@ spec:
     {{- end }}
     port: https
     scheme: https
+{{- if .Values.kubeApiServer.serviceMonitor.metricRelabelings }}
+    metricRelabelings:
+{{ toYaml .Values.kubeApiServer.serviceMonitor.metricRelabelings | indent 6 }}
+{{- end }}
 {{- if .Values.kubeApiServer.relabelings }}
```

Comment:
In the `README.md` you reference this as `kubeApiServer.serviceMonitor.relabelings`